deploy-openshift/apps/expense/post_expense.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if [ $# -le 1 ]
+  then
+    echo "Incorrect arguments provided"
+    echo "    Usage: $0 HOST AMOUNT"
+    echo ""
+    echo "    Example: $0 localhost:8080 10"
+    exit 1
+fi
+
+HOST=$1
+AMOUNT=$2
```

Comment:
Let's always quote vars (see e.g. [here](https://google.github.io/styleguide/shellguide.html#variable-expansion))

```suggestion
HOST="${1}"
AMOUNT="${2}"
```